src/scripts/pokemons/PokemonList.ts

```diff
@@ -8087,7 +8087,7 @@ const pokemonList = createPokemonArray(
         'levelType': LevelType.mediumfast,
         'exp': 158,
         'catchRate': 75,
-        'evolutions': [new WeatherEvolution('Cherrim', 'Cherrim (sunshine)', WeatherType.Sunny)],
+        'evolutions': [new WeatherRestrictedLevelEvolution('Cherrim (overcast)', 'Cherrim (sunshine)', 1, [WeatherType.Sunny])],
```

Comment:
Would it also be worth adding `Cherrim (sunshine)` as a possible weather restricted level 25 evolution for `Cherubi`?
And `Cherrim (overcast)` as a possible evolution for `Cherrim (sunshine)`?

Reply:
Oh wait, nvm, I just noticed that `Cherrim (overcast)` is not weather dependent, it's just a level 25 evolution. 😅